### DISCUSSION TITLE
Conditional Alerts?

### DISCUSSION THREAD

**Discussion by Miquest** [NightscoutFoundation/xDrip#2772] [Ideas]
Hi!
I just replaced my Dexcom G6 sensor and the readings are mostly not really precise in the first 24 hours. 
Wouldn't it be a helpful feature if A special alarm profile would activate within the first 24 hours of sensor runtime, where these inaccuracies be taken into account?

I know it's just a "gimmick", but I also think that it could be helpful for many users

**Comment by Navid200** [comment]
This is something that varies from person to person.

Therefore, I don't see what such an alert would be about and how it would function.

**Reply by Miquest** [reply]
Okay, maybe I expressed myself wrongly. In the first 24 hours, CGM systems are often relatively imprecise. This isn't always the case, but most of the time. Both the Dexcom and the Libre due to the wound reaction.  My idea was that you can set an optional alarm profile to be activated when xDrip detects that the sensor is less than 24 hours old. In this way one could counteract these inaccuracies.

**Reply by Navid200** [reply]
Let's say we add such an alert.
So, you will get an alert that your sensor is younger than 24 hours. 
What would you do then?

**Reply by Miquest** [reply]
No alert if the sensor is younger than 24 hours. 
Let's say, I'm warned if my BZ goes under 80 mg/dl or over 180 mg/dl. If my sensor is younger than 24 hours, xDrip should warn already if it goes under 90 mg/dl and over 200 mg/dl, because the sensor is currently still measuring too high. Currently, I set the alerts on a new value manually.
